.github/workflows/coffee.yml

```diff
@@ -9,8 +9,7 @@ jobs:
   coffee-break:
     runs-on: ubuntu-latest
     container:
-      image: quay.io/redhat-appstudio-qe/qe-tools:latest
-
+      image: quay.io/konflux-ci/qe-tools:latest
```

Comment:
we need to move that image to quay.io/konflux-ci/qe-tools before we merge this PR. 

could you run
```
oc image mirror quay.io/redhat-appstudio-qe/qe-tools:latest quay.io/konflux-ci/qe-tools:latest
```

and also add admin permission to konflux qe folks for quay.io/konflux-ci/qe-tools please?

Reply:
The image is mirrored and I requested team members' username. @psturc Is it ok for you to merge?